Packs/Zabbix/Integrations/Zabbix/Zabbix.yml

```diff
@@ -0,0 +1,496 @@
+        type: date
+      - contextPath: Zabbix.host_get.flags
+        description: Origin of the host.
+        type: number
+      - contextPath: Zabbix.host_get.inventory_mode
+        description: (writeonly) Host inventory population mode.
+        type: number
+      - contextPath: Zabbix.host_get.ipmi_authtype
+        description: IPMI authentication algorithm.
+        type: number
+      - contextPath: Zabbix.host_get.ipmi_available
+        description: Availability of IPMI agent.
+        type: number
+      - contextPath: Zabbix.host_get.ipmi_disable_until
+        description: The next polling time of an unavailable IPMI agent.
+        type: date
+      - contextPath: Zabbix.host_get.ipmi_error
+        description: Error text if IPMI agent is unavailable.
+        type: string
+      - contextPath: Zabbix.host_get.ipmi_errors_from
+        description: Time when IPMI agent became unavailable.
+        type: date
+      - contextPath: Zabbix.host_get.ipmi_password
+        description: IPMI password.
+        type: string
+      - contextPath: Zabbix.host_get.ipmi_privilege
+        description: IPMI privilege level.
+        type: number
+      - contextPath: Zabbix.host_get.ipmi_username
+        description: IPMI username.
+        type: string
+      - contextPath: Zabbix.host_get.jmx_available
+        description: Availability of JMX agent.
+        type: number
+      - contextPath: Zabbix.host_get.jmx_disable_until
+        description: The next polling time of an unavailable JMX agent.
+        type: date
+      - contextPath: Zabbix.host_get.jmx_error
+        description: Error text if JMX agent is unavailable.
+        type: string
+      - contextPath: Zabbix.host_get.jmx_errors_from
+        description: Time when JMX agent became unavailable.
+        type: date
+      - contextPath: Zabbix.host_get.maintenance_from
+        description: Starting time of the effective maintenance.
+        type: date
+      - contextPath: Zabbix.host_get.maintenance_status
+        description: Effective maintenance status.
+        type: number
+      - contextPath: Zabbix.host_get.maintenance_type
+        description: Effective maintenance type.
+        type: number
+      - contextPath: Zabbix.host_get.maintenanceid
+        description: ID of the maintenance that is currently in effect on the host.
+        type: string
+      - contextPath: Zabbix.host_get.name
+        description: Visible name of the host.
+        type: string
+      - contextPath: Zabbix.host_get.proxy_hostid
+        description: ID of the proxy that is used to monitor the host.
+        type: string
+      - contextPath: Zabbix.host_get.snmp_available
+        description: Availability of SNMP agent.
+        type: number
+      - contextPath: Zabbix.host_get.snmp_disable_until
+        description: The next polling time of an unavailable SNMP agent.
+        type: date
+      - contextPath: Zabbix.host_get.snmp_error
```

Comment:
The convention is to call the outputs with the name of the object they return, and not the command name.
In this case
```
Zabbix.Host. 
Zabbix.Trigger.
```
instead of 
```
Zabbix.host_get
Zabbix.trigger_get
```